QMLComponents/boundcontrols/boundcontrolterms.cpp

```diff
@@ -297,8 +298,8 @@ Terms BoundControlTerms::_getTermsFromOptions(const Json::Value& option) const
 	Json::Value valueOption = _isValueWithTypes(option) ? option["value"] : option;
```

Comment:
Maybe we should also consider making a proper Option type again?

Reply:
The BoundControl classes replace the old Option classes.

Reply:
Yes, the name is is not the best one, I agree. Maybe it should be back to Option